python/ray/util/client/api.py

```diff
@@ -247,6 +253,18 @@ def get_runtime_context(self):
     def get_gpu_ids(self) -> list:
         return []
 
+    def timeline(self, filename: Optional[str] = None) -> Optional[List[Any]]:
+        logger.info("Timeline will include events from other clients using "
+                    "this server.")
+        import ray.core.generated.ray_client_pb2 as ray_client_pb2
```

Comment:
circular dependency?

Reply:
Yeah, let me add the comment that is used elsewhere in this file.